### PR TITLE
Set an 'engine' in package.json for npm v2

### PR DESCRIPTION
**Pull request by magopian**
Does the same as https://github.com/mozilla-services/react-jsonschema-form/issues/55

r? @n1k0

### CODE REVIEW

**Comment by magopian** [comment]
This is a fail, PR on the wrong projet :'(

**Comment by n1k0** [comment]
Please reopen, this project needs love too ;)

File: package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
     "bin",
     "dist"
   ],
+  "engineStrict": false,
+  "engines" : { "npm" : "^2.14.7" },
```
Comment (n1k0) [inline package.json]: Please add a requirement for node >= 4

**Comment by n1k0** [comment]
r+wc

**Comment by magopian** [comment]
Added the requirement for node, what do you think @n1k0 ?

**Comment by n1k0** [comment]
r+ !

**Comment by n1k0** [comment]
While you're now our npm engine requirements expert, would you mind keeping doing this for Kinto.js? :D

**Comment by magopian** [comment]
Done: https://github.com/Kinto/kinto.js/pull/360

**Comment by n1k0** [comment]
Thanks buddy :)

**Comment by Natim** [comment]
Great :+1:

**Pull request merged**